docker/Dockerfile

```diff
@@ -18,8 +18,8 @@ ENV SRC_DIR /kubo
 WORKDIR $SRC_DIR
 
 # Install the plugin and build ipfs
-RUN go get github.com/ipfs/go-ds-s3/plugin@latest
-RUN echo "\ns3ds github.com/ipfs/go-ds-s3/plugin 0" >> plugin/loader/preload_list
+RUN go get github.com/ipfs/go-ds-s3/go-ds-s3-plugin@latest
+RUN echo "\ns3ds github.com/ipfs/go-ds-s3/go-ds-s3-plugin 0" >> plugin/loader/preload_list
```

Comment:
Is that even working ? I thought it tried it and it refused to import it because it was `main`.
It is still `main`, what I expect to happen is `github.com/ipfs/go-ds-s3/plugin` would be imported.

Reply:
Tbh I just modified the path the same as I did in README.md
Creating the Docker Image is broken anyway because docker image `ipfs/kubo v0.19.0` was not compatible with latest ipfs version (kubo/bin/container_daemon dependencies changed).

Following commit will fix docker image: [`b37ea16`](https://github.com/ipfs/go-ds-s3/pull/275/commits/b37ea1645738f04dd76f5e7ceaebca6d273d76ba)

For long term: I guess docker image is to be thought through because ipfs version is hard coded and parameter are unclear and incomplete (f.E. regionEndpoint is missing). But imo this is to be done in a separate PR. Let me know if you share this opinion.